.nvmrc

```diff
@@ -1 +1 @@
-v16.13.0
\ No newline at end of file
+20
```

Comment:
We still use Vercel as a default. In Vercel you can go up to v18. Node20 is in beta